src/eval-frame/components/reps/__stories__/reps.stories.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+
+import { storiesOf } from "@storybook/react";
+
+// need to manually load css for react table
+// but even with this some styles seem to be missing
+import "../../../../../node_modules/react-table/react-table.css";
+
+import {
+  allCases,
+  rowTableCases
+} from "../__test_helpers__/reps-test-value-cases";
+import {
+  tinyRepSerializer,
+  getClass,
+  getType,
+  objSize
+} from "../rep-utils/tiny-rep-serializer";
+
+import tinyRep from "../tiny-reps";
+
+import ValueRenderer from "../value-renderer";
+
+import TableRenderer from "../data-table-rep";
+
+const allTinyReps = storiesOf("all test cases", module);
+allTinyReps.add("table of type/class info", () => {
+  return (
+    <table>
+      {Object.entries(allCases).map(caseNameAndVal => {
+        const [name, value] = caseNameAndVal;
+        return (
+          <tr key={name}>
+            <td>{name}</td>
+            <td>{getType(value)}</td>
+            <td>{getClass(value)}</td>
+            <td>{objSize(value)}</td>
+          </tr>
+        );
+      })}
+    </table>
+  );
+});
+
+allTinyReps.add("all the tiny reps", () => {
+  return (
+    <table>
+      {Object.entries(allCases).map(caseNameAndVal => {
+        const [name, value] = caseNameAndVal;
+        return (
+          <tr key={name}>
+            <td>{name}</td>
+            <td>{tinyRep(tinyRepSerializer(value))}</td>
```

Comment:
my comments re: the data table & using `<TinyRep value={value} />` apply here, but additionally it would be great to put the non-tiny rep here for comparison as well.

Reply:
addressed `<TinyRep value={value} />` in the other thread;

re the "non-tiny" rep -- do you mean the normal react-inspector? in any case, this story is really meant to just show the tiny-reps. also, including the react-inspector reps would probably blow up the page since some of the things here are e.g. giant strings that react-inspector does not handle well at all. (all of the standard reps are available in their own stories to prevent that)

Reply:
Yep that's what I mean. I suppose down the line as others work on the reps and use the storybook to get acclimated, they know what they're comparing things to, especially if we end up rolling our own longer reps. I understand they'd definitely blow up on some of the longer ones as is, so probably not worth adding them there, and I imagine the point of the current test set is to show how big objects / primitives boil down. Maybe the solution is to add another storybook page for a subset of sensible data values so comparison is easier. This all said, doesn't have to be done in this PR if you think it's not really worthwhile!